.github/workflows/generate-command-reference.yml

```diff
@@ -0,0 +1,29 @@
+name: Automatically generate command reference topics
+on:
+  push:
+  workflow_dispatch:
+
+env:
+  AMC_COMMAND_REFERENCE_DIR: "./reference/cmd-ref/amc-command-reference"
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-24.04
+    steps:
+    - name: Setup Anbox Cloud
+      uses: canonical/anbox-cloud-github-action@main
```

Comment:
We should pin to a specific git revision and let renovate take care of updating when there are changes.